test_jsons.py

```diff
@@ -15,6 +17,17 @@ def test_zip_poverty():
     #tests zip json file
     check_file('final_jsons/acs5zippoverty_output.json')
 
+def test_county_child_poverty():
+    #runs county script to create/update county json
+    county_child_poverty_script()
+    #tests county json file
+    check_file('final_jsons/acs5county_child_poverty_output.json')
+    
+def test_zip_child_poverty():
+    #runs county script to create/update county json
+    zip_child_poverty_script()
+    #tests county json file
+    check_file('final_jsons/acs5zip_child_poverty_output.json')    
 
```

Comment:
Could you merge this into your other PR?

Reply:
Not just that portion in particular. We can do one PR

Reply:
Can they not be merged separately?

Reply:
They can but if they conflict, that conflict will need to be resolved before the other merges. If it is in one PR, then we know it does not conflict.